WindowsServerDocs/administration/server-manager/run-best-practices-analyzer-scans-and-manage-scan-results.md

```diff
@@ -14,22 +14,22 @@ ms.date: 10/16/2017
 
 In Windows management, *best practices* are guidelines that are considered the ideal way, under typical circumstances, to configure a server as defined by experts. For example, it is considered a best practice for most server applications to keep open only those ports required for the applications to communicate with other networked computers, and block unused ports. Although best practice violations, even crucial ones, are not necessarily problematic, they indicate server configurations that can result in poor performance, poor reliability, unexpected conflicts, increased security risks, or other potential problems.
 
-Best Practices Analyzer (BPA) is a server management tool that is available in Windows Server 2012 R2,  Windows Server 2012, and  Windows Server 2008 R2. BPA can help administrators reduce best practice violations by scanning roles that are installed on managed servers that are running  Windows Server 2012  or  Windows Server 2008 R2, and reporting best practice violations to the administrator.
+Best Practices Analyzer (BPA) is a server management tool that is available in Windows Server 2019, Windows Server 2016, Windows Server 2012 R2,  Windows Server 2012, and  Windows Server 2008 R2. BPA can help administrators reduce best practice violations by scanning roles that are installed on managed servers that are running a supported version of, and reporting best practice violations to the administrator.
```

Comment:
I am fairly certain we can simplify the sentence further by skipping "that is", which is an overused "filler" expression in the English language.
```suggestion
Best Practices Analyzer (BPA) is a server management tool available in Windows Server 2019, Windows Server 2016, Windows Server 2012 R2,  Windows Server 2012, and  Windows Server 2008 R2. BPA can help administrators reduce best practice violations by scanning roles that are installed on managed servers that are running a supported version of, and reporting best practice violations to the administrator.
```

Reply:
Looks Fine to me